lib/features/email/presentation/widgets/calendar_event/event_time_information_widget.dart

```diff
@@ -9,17 +9,19 @@ import 'package:tmail_ui_user/main/localizations/app_localizations.dart';
 class EventTimeInformationWidget extends StatelessWidget {
 
   final String timeEvent;
+  final bool isFree;
 
   const EventTimeInformationWidget({
     super.key,
-    required this.timeEvent
+    required this.timeEvent,
+    required this.isFree,
```

Comment:
```suggestion
    this.isFree = true,
```

To avoid charging more than where it's called.

Reply:
Not necessary. At this moment there is only 1 reference.

Reply:
I see more changes here:

![Screenshot 2025-04-02 at 13 51 21](https://github.com/user-attachments/assets/2d7ac2ff-30e3-4ff2-a029-cb82a4520998)

Reply:
That is from `BlobCalendarEvent` class, not `EventTimeInformationWidget`. Would making `isFree` of `BlobCalendarEvent` optional help?

Reply:
Reverted to non-required